src/tokens.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+  --ds-auro-popover-arrow-boxshadow-color: rgb(0 0 0 / 0.08);
+  --ds-auro-popover-boxshadow-color: rgb(0 0 0 / 0.08), 0 2px 5px 1px rgb(0 0 0 / 0.08);
```

Comment:
Why is the color in here twice? And why the offset numbers?

Also, this should be changed to use the closest matching black opacity token in the new tokens.